lib/tasks/osc.rake

```diff
@@ -74,6 +74,21 @@ namespace :osc do
     end
   end
 
+  def check_IDs_in_changes_file!
+    # Checking if any new bugzilla,fate,... IDs are defined
+    # in the *.changes file
+    puts "Checking IDs in *.changes file" if verbose
+    Dir.chdir(osc_checkout_dir) do
+      cmd = "osc -A '#{obs_api}' diff *.changes | grep \"bnc#[0-9]\\|fate#[0-9]\\|boo#[0-9]\\|bsc#[0-9]\\|bgo#[0-9]\""
+      puts cmd if verbose
+      `bash -c '#{cmd}'`
+      return if $?.success?
+    end
+    puts "Stopping, missing new bugzilla or fate entry in the *.changes file"
+    puts "Valid entries are bnc#<number>, fate#<number>, boo#<number>, bsc#<number>, bgo#<number>"
```

Comment:
cve has everytime a bugzilla entry because these bugzilla entries are generated by the security team.  I think that these bugzilla entries are more important. That's why I have not added cve. But I can add it too. Case insensitive is a good point.
Plugins: That has been my first approach. But I have had a lot of problems with installing these plugins. There are some missing pythons dependencies there and as far I see the plugin is not packaged. So from my point of view the environment is too complex here. I think the packaging_tasks gem should as most independent as possible. It is not worth for a single line of code (grep the IDs). But feel free to overrule my self :-)